apps/breethe/test/support/factories.ex

```diff
@@ -18,7 +18,7 @@ defmodule Breethe.Factory do
   def measurement_factory do
     %Breethe.Data.Measurement{
       parameter: "pm10",
-      measured_at: DateTime.from_naive!(~N[2019-01-01 00:00:00.000000], "Etc/UTC"),
+      measured_at: DateTime.utc_now(),
```

Comment:
I think what we want is to pass this date in from the test so that we know exactly what it is going to be and can assert on the value.

Reply:
Do you think it's unclear if it is in the factory? The assumption is that any location/measurement inserted will have the current UTC. I would be more in favour of keeping the default as is and specifying in the `insert(...)` only when using a different time? Also I think this
```elixir
      measured_at =
        location.last_updated
        |> Map.put(:microsecond, {0, 0})
        |> DateTime.to_iso8601()
```
makes the information available if need be as it explicitly points to the factory and the time isn't really relevant to a high-level understanding of the test nor the main point of the tests?

Reply:
I think the value that's declared in the factory doesn't really matter much - the thing is we shouldn't have assertions in the tests on values that are defined in the factories as that's very hard to follow. See Andy's post on the topic here: https://simplabs.com/blog/2018/12/20/factories-best-practices.html. The thing with a fixed date is that it is either in the past in which case the value would be too old and thus not be read or in the future which is something that can never happen in reality. I think what we really want is a sensible default which would be something like `Time.now - 2.hours`

Reply:
Ok, I'll make the changes. I don't really understand why we should use `Time.now - 2.hours` rather than `Time.now` though?

Reply:
A measurement from 2 hours ago is just more likely than one from right at the moment when you open the app - it doesn't make a big difference but you'd always aim for reasonably _"realistic"_ default values in factories.